src/nifgen/examples/nifgen_script.py

```diff
@@ -21,7 +21,7 @@
 RAMP_DOWN = [-1.0 * x for x in RAMP_UP]
 SQUARE_WAVE = [1.0 if x < (NUMBER_OF_SAMPLES / 2) else -1.0 for x in range(NUMBER_OF_SAMPLES)]
 SAWTOOTH_WAVE = RAMP_UP[::2] + [(-1 + x) for x in RAMP_UP][::2]
-GAUSSIAN_NOISE = [random.gauss(0, 0.2) for x in range(NUMBER_OF_SAMPLES)]
+GAUSSIAN_NOISE = [sorted([-1.0, random.gauss(0, 0.2), 1.0])[1] for x in range(NUMBER_OF_SAMPLES)]
```

Comment:
By clipping the waveform, we are no longer generating noise with a proper gaussian distribution. That makes the example incorrect.

I think we're better off removing the gaussian noise option altogether, rather than being almost but not really correct. This also allows us to stop importing gaussian.

Reply:
>By clipping the waveform, we are no longer generating noise with a proper gaussian distribution.

Good point.